### PR TITLE
fix #78510 hotstar.com

### LINKED ISSUE #78510: www.hotstar.com
### Issue URL (Incorrect Blocking)
 [https://www.hotstar.com/in](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.hotstar.com%2Fin)
### Comment
Username: @Aarjav80
> Hotstar is not working properly with Adguard enabled. I think it is pretty clear from screenshots.

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Block WebRTC, <br>Block Flash, <br>Hide your Referrer from third-parties (www.hotstar.com)
DNS filtering: | server: `System`<br>
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Fanboy's Annoyances, <br/>AdGuard Annoyances
Userscripts: | https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js,<br>https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
Other extensions used: | Dashlane, Adguard VPN, Adode acrobat, Kaspersky Internet Security, Grammarly and Sponsor block by ajay.

### PR DESCRIPTION
**Pull request by wout-codes**
#78510

### CODE REVIEW

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -79,6 +79,8 @@ semprotyuk.com##div[style^="margin-bottom:150px;"][style*="padding-right:2em;"]
 ||n11scdn.akamaized.net/assets/javascripts/ha.js
 ||playbrain.io/analytics/
 ||linkwithin.com/pixel.png
+||hotstar.com^*/track
+||hotstar.com^*/identify
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:83]: better`/*` if no `:port_number`

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -305,6 +305,9 @@ bookriot.com##.inside-content-ad-container
 bookriot.com##.widget_random_content_pro_widget
 123movies.*##.row-btns-special > a[target="_blank"]
 google.*##.cu-container
+hotstar.com##.masthead-area
+last.fm##.full-bleed-ad-container
+last.fm##.tonefuze
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt:310]: These rules - your, not mistake?

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt]: Bad merge.

File: SpywareFilter/sections/specific.txt [hunk 3]
```diff
@@ -79,6 +79,8 @@ semprotyuk.com##div[style^="margin-bottom:150px;"][style*="padding-right:2em;"]
 ||n11scdn.akamaized.net/assets/javascripts/ha.js
 ||playbrain.io/analytics/
 ||linkwithin.com/pixel.png
+||hotstar.com^/*track
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt:82]: No, I mean like that `||hotstar.com/*/track`

Comment (wout-codes) [reply SpywareFilter/sections/specific.txt:82]: Oh yeah.

**Review by Alex-302 — APPROVED** [review]
Thanks.

**Pull request merged**